README.md

```diff
@@ -1,2 +1,43 @@
 # readium-css
-Place of creation of the reference CSS for readium projects, beginning with Readium-2.
+
+A set of reference stylesheets for EPUB Reading Systems, starting with Readium 2.
+
+Readium CSS provides: 
+
+- a CSS normalize for EPUB contents;
+- paged and scrolled views;
+- default styles;
+- reading modes (night, sepia, etc.);
+- themes;
+- user settings;
+- media overlays and user highlights.
+
+## License
+
+BSD-3-Clause (http://opensource.org/licenses/BSD-3-Clause)
+
+See [license.txt](https://github.com/readium/readium-css/blob/master/LICENSE).
+
+## Scope of this project
+
+The major goal of Readium CSS is to provide implementers with reliable and modern styles for EPUB 2 and EPUB 3 publications, and authors with good interoperability. 
```

Comment:
isn't it "... provide implementers and authors with... publications, with good interoperability." ?